src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -468,7 +415,35 @@ def get_job_logs(
                 "lines": logs,
             }
         except Exception as e:
-            raise transform_job_exception(e, "Unable to retrieve job logs")
+            return {
+                "job_id": job.job_id,
+                "batch_id": job.batch_id,
+                "error": e.message,
+            }
+
+    def get_job_logs_for_list(
+        self,
+        job_id_list: List[str],
+        first_line: int = 0,
+        num_lines: int = None,
+        latest: bool = False,
+    ) -> dict:
+        """
+        Fetch the logs for a list of jobs. Note that the parameters supplied are applied to all jobs.
+        """
+        job_ids, error_ids = self._check_job_list(job_id_list)
+
+        output = {}
+        for job_id in job_ids:
+            output[job_id] = self.get_job_logs(job_id, first_line, num_lines, latest)
+
+        for error_id in error_ids:
+            output[error_id] = {
+                "job_id": error_id,
+                "error": DOES_NOT_EXIST,
+            }
+
+        return output
```

Comment:
simple wrapper for applying a job-related method to a list of job IDs, collecting errors and valid output